app/javascript/dashboard/routes/dashboard/helpcenter/components/ArticleSearch/Header.vue

```diff
@@ -1,53 +1,60 @@
-<script>
-import keyboardEventListenerMixins from 'shared/mixins/keyboardEventListenerMixins';
+<script setup>
+import { ref, onMounted } from 'vue';
+import { useKeyboardEvents } from 'dashboard/composables/useKeyboardEvents';
 
-export default {
-  name: 'ChatwootSearch',
-  mixins: [keyboardEventListenerMixins],
-  props: {
-    title: {
-      type: String,
-      default: 'Chatwoot',
-    },
-  },
-  data() {
-    return {
-      searchQuery: '',
-      isInputFocused: false,
-    };
-  },
-  mounted() {
-    this.$refs.searchInput.focus();
+defineProps({
+  title: {
+    type: String,
+    default: 'Chatwoot',
   },
-  methods: {
-    onInput(e) {
-      this.$emit('search', e.target.value);
-    },
-    onClose() {
-      this.$emit('close');
-    },
-    onFocus() {
-      this.isInputFocused = true;
-    },
-    onBlur() {
-      this.isInputFocused = false;
+});
+
+const emit = defineEmits(['search', 'close']);
+
+const articleSearchHeaderRef = ref(null);
+const searchInputRef = ref(null);
+const searchQuery = ref('');
+const isInputFocused = ref(false);
```

Comment:
Are we using the `isInputFocused` anywhere?

Reply:
@scmmishra Yeah, we are not using removed it. https://github.com/chatwoot/chatwoot/pull/9897/commits/3436f846b58048be3e76eb2ef96865182332f4ba